kernel/Makefile

```diff
@@ -51,9 +53,12 @@ sign_$(2): build_$(2)
 		 docker tag linuxkit/$(IMAGE):$(1)-$(HASH) linuxkit/$(IMAGE):$(2) && \
 		 DOCKER_CONTENT_TRUST=1 docker push linuxkit/$(IMAGE):$(2))
 
+update-config_$(2): build_$(2)
+	docker run linuxkit/$(IMAGE):$(1)-$(HASH) cat /kernel_config > kernel_config-$(2)
```

Comment:
I'm not sure about this. What this does is it builds the kernel docker image and then copies the kernel config out (which is nice to have). But if we then commit it, this will change the git tree hash and thus we have to rebuild, right? So as a developer, I'd have to do a full kernel compile when a download kernel, apply patches, copy config and `make defconfig && make oldconfig` would suffice. I think, I'd prefer having a separate Dockerfile to update the kernel config for quicker turnaround